lib/gui/components/update-notifier/controllers/update-notifier.js

```diff
@@ -16,7 +16,9 @@
 
 'use strict';
 
-module.exports = function($uibModalInstance, SettingsModel, AnalyticsService, UPDATE_NOTIFIER_SLEEP_DAYS, options) {
+const SettingsModel = require('../../../models/settings');
```

Comment:
Can we name the object just `settings` now? The uppercase first letter and the `Model` suffix are Angular-ish conventions, so we should use node naming conventions now.